Systems/c172p-damage.xml

```diff
@@ -288,20 +289,14 @@
             <test logic="AND" value="0.12">
                 /velocities/airspeed-kt GT damage/limits-vne-low
                 damage/roll-moment GT 4000.0
-                wing-damage/left-wing EQ 0.0
+                wing-damage/left-wing LT 0.12
             </test>
 
             <!-- Over-g -->
             <test logic="AND" value="0.12">
-                damage/force-aero-lbs GT damage/limits-lift-medium
```

Comment:
The block that I removed does the same effect as the `Over-g `block under `Heavy damage` (line 267), except for 0.12 instead of 0.3. My initial idea has been damage 0.12 in asymmetric conditions (break a single wing, the aircraft is still controllable in roll, although uneasy). And 0.3 for symmetric damage, the aircraft can still fly but with a serious loss of lift.